resources/macos_user.rb

```diff
@@ -44,6 +44,7 @@ def setup_assistant_keypair_values
       'LastSeenCloudProductVersion' => node['platform_version'],
       'LastPreLoginTasksPerformedVersion' => node['platform_version'],
       'LastPreLoginTasksPerformedBuild' => node['platform_build'],
+      'LastPrivacyBundleVersion' => '2',
```

Comment:
@jazaval This is not necessarily related to this PR, but note that Chef has wrapped [Core Foundations](https://developer.apple.com/documentation/corefoundation) in a Ruby class via FFI. This allows you to set preferences similarly to what is done in the [macos_userdefaults](https://github.com/chef/chef/blob/main/lib/chef/resource/macos_userdefaults.rb#L128) resource:

`CF::Preferences.set!(new_resource.key, nil, new_resource.domain, to_cf_user(new_resource.user), to_cf_host(new_resource.host))`

I'm not sure if this would afford any additional utility over the current method, but it might be handy to note its existence.